### ISSUE TITLE
App Manager - Choosing The Filename

### ISSUE THREAD

**Issue by HonestAbe7** [TeamAmaze/AmazeFileManager#3331]
I would like to be able to specify what the filename for my backed up apps would be from the App Manager. I prefer the app name and version. Thanks

**Comment by shuvashish76** [comment]
> I would like to be able to specify what the filename for my backed up apps would be from the App Manager.

I don't think it's necessary as you could change the apk file name from FM after backup the apk file. 

> I prefer the the ... version.

In favour of backup apk files with **version code**.

**usecase** : backup multiple version apk files from the same package ID
**current backup format** : "`app name_package name`"
**current issue** : backup an app > uninstall > install same package but different version > backup new installed version > Amaze replaces the previous version apk file with new version
**requested backup format** : "`app name_package name_version code`"

Similar issue #2069

**Comment by HonestAbe7** [comment]
True. I just like to know the version in case I need to roll back versions. I really don't care about the package; just the app name and optionally the version.

**Comment by shuvashish76** [comment]
> I really don't care about the package; just the app name and optionally the version

I agree but when we report an issue we've to think from other users' point of view, that's how OSS works. I think package id is important for some users...

> The App Manager is way too complicated to figure out.

Too many features- I agree, complicated- I don't think so...
Everything is nicely documented [here](https://muntashirakon.github.io/AppManager/en/) or instruments page from AM. Or ask your queries on their [discussion forum](https://github.com/MuntashirAkon/AppManager/discussions)

**Comment by HonestAbe7** [comment]
Thanks for the response. I found an app on F-Droid that does exactly what I want. It is called APK Kit.
